### ISSUE TITLE
[Clock] calendar with CJK locale: mix of fullwidth and halfwidth characters causing date column shift

### ISSUE THREAD

**Issue by ishigoya** [Alexays/Waybar#976]
In the calendar, when the day of the week is represented by a fullwidth character, the date columns are shifted to the left. It looks as if the character separating the days of the week is also fullwidth, while the character separating the date columns is halfwidth. 

Here's a picture:

![image](https://user-images.githubusercontent.com/16798645/104420170-d0c95480-55bc-11eb-8477-f9e1c38b797f.png)

.. and my clock config:

~~~
"clock": {
    "timezone": "Asia/Tokyo",
    "locale": "ja_JP.UTF-8",
    "format": "{:%Om月%Od日（%a）%H%M}",
    "tooltip-format": "<big>{:%Y %B}</big>\n<tt><small>{calendar}</small></tt>",
}
~~~

Possibly related to #910 ?

**Comment by nullobsi** [comment]
i've submitted a PR that should fix this issue (though the whitespace rendering may have issues for some reason) :)

**Comment by Alexays** [comment]
Closed by #1007
